loadbalancer/algorithm_test.go

```diff
@@ -316,13 +317,14 @@ func TestConsistentHashBoundedLoadSearch(t *testing.T) {
 	}})[0]
 	ch := route.LBAlgorithm.(*consistentHash)
 	ctx := &routing.LBContext{
-		Request:  r,
-		Route:    route,
-		Params:   map[string]interface{}{ConsistentHashBalanceFactor: 1.25},
-		Registry: routing.NewEndpointRegistry(routing.RegistryOptions{}),
+		Request:   r,
+		Route:     route,
```

Comment:
Consider passing only required fields instead of the full `routing.Route` struct. This struct has `LBEndpoints` field which created confusion between this field and `routing.LBContext.Endpoints` field.